exceptional_synchrony.gemspec

```diff
@@ -1,20 +1,24 @@
 require File.expand_path('../lib/exceptional_synchrony/version', __FILE__)
 
-Gem::Specification.new do |gem|
-  gem.name        = 'exceptional_synchrony'
-  gem.date        = '2014-01-16'
-  gem.summary     = 'Extensions to EventMachine/Synchrony to work well with exceptions'
-  gem.description = %q{}
-  gem.authors     = ['Colin Kelley']
-  gem.email       = 'colin@invoca.com'
-  gem.homepage    = 'https://github.com/Invoca/exceptional_synchrony'
-  gem.license     = 'MIT'
-  gem.files       = `git ls-files`.split($/)
-  gem.version     = ExceptionalSynchrony::VERSION
+Gem::Specification.new do |spec|
+  spec.name        = 'exceptional_synchrony'
+  spec.date        = '2014-01-16'
+  spec.summary     = 'Extensions to EventMachine/Synchrony to work well with exceptions'
+  spec.description = %q{}
+  spec.authors     = ['Invoca']
```

Comment:
I'm fine if you want to leave you name on it. Do we have a standard for our open source gems?